### ISSUE TITLE
"Undefined variable: b:parinfer_last_changedtick" when working in narrowed buffer

### ISSUE THREAD

**Issue by lxsli** [eraserhd/parinfer-rust#28]
I just found this awesome plugin: https://github.com/chrisbra/NrrwRgn

It works great. Unfortunately when I `set ft=clojure` in the narrowed buffer, parinfer-rust throws an exception any time I move the cursor:

```
Error detected while processing function <SNR>46_process:                                                                                                                                                                                                                                                                                                                                   
line    4:                                                                                                                                                                                                                                                                                                                                                                                  
E121: Undefined variable: b:parinfer_last_changedtick                                                                                                                                                                                                                                                                                                                                       
Error detected while processing function <SNR>46_process:                                                                                                                                                                                                                                                                                                                                   
line    4:                                                                                                                                                                                                                                                                                                                                                                                  
E15: Invalid expression: b:parinfer_last_changedtick != b:changedtick                                                                                                                                                                                                                                                                                                                       
```

**Comment by lxsli** [comment]
FWIW the narrowed region is pure Clojure code, with two top-level forms. There's no ns declaration, if that matters?

Aha - all this also happens if I just create a new split, paste the code and `set ft=clojure`. Nothing to do with the new plugin.

**Comment by eraserhd** [comment]
@lxsli hi!  Have you updated recently?  I just recently merged a fix for a similar issue, and I would imagine it would fix this as well.

Let me know.

**Comment by eraserhd** [comment]
(This was resolved with #27 )

**Comment by lxsli** [comment]
Confirmed, thanks!
